CHANGELOG.md

```diff
@@ -10,6 +10,78 @@ PRs and issues are linked, so you can find more about it. Thanks to [ChangelogLi
 
 <!-- changelog-linker -->
 
+## Unreleased
+
+### Added
+
+- [#4779] [CI] Add PHP 8.0 to Github actions tests, Thanks to [@samsonasik]
+- [#4791] [CodeQuality] Add DateTimeToDateTimeInterfaceRector, Thanks to [@simivar]
+- [#4789] [DX] Add init-recipe command to create recipe config in root
+- [#4740] [Docs] Add docblock docs
+- [#4762] [Downgrade] [PHP 8.0] Add DowngradePropertyPromotionToConstructorPropertyAssignRector
+- [#4788] [PHP 8.0] [Downgrade] Add DowngradeNonCapturingCatchesRector
+- [#4787] [PHP 8.0] [Downgrade] Add DowngradeNonCapturingCatchesRector
+- [#4780] [PHPStanExtensions] Add ForbiddenMethodCallOnTypeRule + move from docComment and Comments to PhpDocInfo
+- [#4741] [PHP 8.0] Add nette utils strings replace, static fixes
+- [#4758] Donwgrade PHP 7.4: Added covariant rule to set, Thanks to [@leoloso]
+- [#4799] add docblock on downgrade by default
+- [#4772] [Downgrade PHP 8.0] Add DowngradeClassOnObjectToGetClassRector, Thanks to [@samsonasik]
+- [#4776] [Downgrade PHP 7.1] Fixes [#4196] Add DowngradePipeToMultiCatchExceptionRector, Thanks to [@samsonasik]
+- [#4764] Added some missing Nette 3.0 tasks, Thanks to [@lulco]
+- [#4792] [PHP 8.0] Add SetStateToStaticRector, Thanks to [@simivar]
+- [#4794] [PHP 8.0] Add FinalPrivateToPrivateVisibilityRector, Thanks to [@simivar]
+- [#4795] [PHP 8.0] Add RemoveParentCallWithoutParentRector to the set config, Thanks to [@simivar]
+
+### Changed
+
+- [#4797] [DX] Use MethodName const instead of string for __set_state, Thanks to [@simivar]
+- [#4782] [DeadDocBlock] decouple return and param rules
+- [#4796] [Downgrade] [PHP 7.3] Downgrade trailing commas in function calls
+- [#4742] [PHP 8.0] [Naming] Skip union type to resolve param name
+- [#4757] [Solid] Skip ChangeAndIfToEarlyReturnRector on nested if in loop, Thanks to [@samsonasik]
```

Comment:
This PR was a effort of several people. Should co-authors also be mentioned in the changelog?

Reply:
This PR was generated and only works with the author. It's not that smart :)

Feel free to update it :+1: